### PR TITLE
Only try to create workers if they exist

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

File: 11_register_hosts.sh [hunk 1]
```diff
@@ -77,4 +77,4 @@ list_workers | make_bm_workers | tee $SCRIPTDIR/ocp/worker_crs.yaml
 
 oc --config ocp/auth/kubeconfig apply -f $SCRIPTDIR/ocp/master_crs.yaml --namespace=openshift-machine-api
 
-oc --config ocp/auth/kubeconfig apply -f $SCRIPTDIR/ocp/worker_crs.yaml --namespace=openshift-machine-api
+[ -f $SCRIPTDIR/ocp/worker_crs.yaml ] && oc --config ocp/auth/kubeconfig apply -f $SCRIPTDIR/ocp/worker_crs.yaml --namespace=openshift-machine-api
```
Comment (dhellmann) [inline 11_register_hosts.sh]: This file is created on line 76 above. What are the conditions you're seeing that lead to it not being created?

Comment (e-minguez) [reply 11_register_hosts.sh]: I don't have any workers in my `bm.json` (I'm thinking about creating the cluster with 3 masters, deploy rook, kubevirt, etc. then add workers eventually)

Comment (dhellmann) [reply 11_register_hosts.sh]: Does that somehow result in the file not being created? Maybe this needs to test if it's empty, instead of whether it exists?

Comment (e-minguez) [reply 11_register_hosts.sh]: You are right, the file is there but empty. I'll fix it, thanks!
